pkg/common/common.go

```diff
@@ -11,7 +11,11 @@ import (
 	"k8s.io/client-go/tools/record"
 )
 
-const UninstallingAnnotation string = "policy.open-cluster-management.io/uninstalling"
+const (
+	UninstallingAnnotation string = "policy.open-cluster-management.io/uninstalling"
+	PolicyIDAnnotation     string = "policy.open-cluster-management.io/policy-compliance-db-id"
```

Comment:
Optional: PolicyDbIdAnnotation or PolicyDbAnnotation recommend. It is little confusing

Reply:
@mprahl  Let me know when you decide to change. I will approve right away!